UnityProject/Assets/Scripts/Gateway/StationGateway.cs

```diff
@@ -78,11 +77,19 @@ protected virtual void UpdateMe()
 				DetectPlayer();
 				timeElapsedServer = 0;
 			}
+
+			timeElapsedServerSound += Time.deltaTime;
+			if (timeElapsedServerSound > 7 && isOn)
+			{
+				DetectPlayer();
+				SoundManager.PlayNetworkedAtPos("machinehum4", Position + Vector3Int.up);
+				timeElapsedServerSound = 0;
+			}
 		}
 		else
 		{
 			timeElapsedClient += Time.deltaTime;
-			if (timeElapsedClient > 1)
+			if (timeElapsedClient > 0.25)
```

Comment:
Why 0.25?

Reply:
Thats the speed of the animation. Too slow at 1 second, so i changed it to 0.25